File: mario_more.c
/**
* CS50 week 1 mario_more
*/

#include <stdio.h>
#include <cs50.h>
int main (void) 
{
int height;
int h=1;
do 
{
printf ("height:");
height=get_int();
} 
while (height<0||height>23);
int i=0;
int t=height;
for (i=0; i<height; i++)
{
int s=0;
for (s=0; s<t-1; s++)
{
    printf(" ");
}
int n=0;
for (n=0; n<h+1; n++)
{
    printf("#");
}
h++;
t--;
    printf("\n");
}
}

File: mario_more.py
#print Mario piramid_less comfortable

def main():

    import cs50

    #get positive integer from a user (the height) greater than 0 lesser than 23
    while (True):
        print("height:", end="")
        height=cs50.get_int()
        if height>0 and height<23:
            break


    width=height-1 #creat a variable to remember amount of spaces to print in each row
    h=1 #create a variable to remember the amount of # to print in each row
    w=1 #create a variable to remember the amount of # to print in second half of double half-pyramid

    #iterate over each row
    for i in range(height):
        #print spaces in each row
        print(" "*width, end="")

        #print # in each row
        print("#"*h, end="")

        h+=1 #increment amount of # to print in the next row
        width-=1 #decrement amout of spaces to print in the next row

        print("  ", end="") #print 2 spaces between halfs of the half-pyramid

        #print second half of the half-pyramid
        print("#"*w, end="")

        w+=1 #increment amount of # in each row of the second half

        print () #print a new line



if __name__ == "__main__":
    main()








